### PR TITLE
CASMINST-5217 fix path issue

### PR DESCRIPTION
**Pull request by dzou-hpe**
# Description

- CASMINST-5217 fix path issue
- CASMINST-5217 update worker rebuild doc

# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

**Review by rustydb — CHANGES_REQUESTED** [review]
Found one area that needs some sprucing up.

```md
troubleshooting/known_issues/SLS_Not_Working_During_Node_Rebuild.md:3:During some node rebuilds (including those that happen during [Stage 1](../../upgrade/1.2/Stage_1.md) and [Stage 2](../../upgrade/1.2/Stage_2.md) of the CSM upgrade process),
```

**Review by rustydb — APPROVED** [review]

**Review by mharding-hpe — APPROVED** [review]

File: operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md [hunk 1]
```diff
@@ -24,21 +24,45 @@ Only follow the steps in the section for the node type that is being rebuilt.
...
+##### Restrictions
+
+In some cases, it is not possible to upgrade all workers in one request. It is system administrator's responsibility to
+make sure that the following conditions are met:
+
+- If the system has more than five workers, then they cannot all be rebuilt with a single request.
+
+    In this case, the rebuild should be split into multiple requests, with each request specifying no more than five workers.
+
+- No single rebuild request should include all of the worker nodes that have DVS running on them.
```
Comment (djlapenta) [inline operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md:52]: @dzou-hpe should we include an example, here, to tell users how to determine which are the 'DVS' nodes?  I don't have that solution, but we could get it and throw it in here?

Comment (djlapenta) [reply operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md:52]: oops, already merged..  I'll follow-up with Di about ^^^..

Comment (rustydb) [reply operations/node_management/Rebuild_NCNs/Rebuild_NCNs.md:52]: Sorry Debbie, this merged before we received your comment. Apologies.

**Pull request merged**